backend/test_chat_with_progress.py

```
@@ -0,0 +1,182 @@
+                        print("   This might be normal if the response doesn't explicitly mention planning")
+                        print("   The important thing is that it's using the SamuraiAgent with planning-first")
+                        
+                        print("\n✅ Chat-with-progress endpoint is working!")
+                        return True
+                        
+                else:
+                    print(f"❌ Request failed with status: {response.status}")
+                    try:
+                        error_text = await response.text()
+                        print(f"   Error: {error_text}")
+                    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)